module-2/backend/week_three.md

```diff
@@ -1,25 +1,41 @@
 ## Week Three Recap
 
 ### Instructions
-Fork this repository. Answer the questions to the best of your ability.
+Fork this repository. Be sure to pull the latest changes to your local repo. Answer the questions to the best of your ability.
 
 Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
 Note: When you're done, submit a PR with a reflection in the comments about how this exercise went for you.
 
 ### Questions
 
-1. What are the 3 main features in an ERD?
-2. Create an ERD for the following database: Restaurants, Customers, Items, Ingredients, Beverages, Orders, Vendors.
-3. What is the entry at the command line to create a new rails app?
-4. What do Models generally inherit from in rails?
-5. What do Controllers generally inherit from in a rails project?
-6. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
-7. What rake task is useful when looking at routes, and what information does it give you?
-8. What is an example of a route helper? When would you use them?
-9. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
-10. What are strong params and why are the necessary?
-11. What role does `form_for` play in helping us create our forms?
-12. How does `form_for` know where to submit the user's input?
-13. Create a form using a `form_for` helper to create a new `Horse`. 
-14. Why do we want to validate our models?
+1. What is the entry at the command line to create a new rails app?
+`rails new SomeName`
+2. What do Models generally inherit from in rails?
+  `ActiveRecord::Base`
+3. What do Controllers generally inherit from in a rails project?
+  `ApplicationController`
+4. How would I create a route if I wanted to see a specific horse in my routes file assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+  `resources :horses, only: [:show]`
+
+5. What rake task is useful when looking at routes, and what information does it give you?
+  `'rake routes', it shows you all of the routes in your application and which pages they route to and the name of the route helper` 
```

Comment:
Like that you're using your own words here. It shows that you're understanding what each piece actually does. Maybe also include the actual headings that rake routes returns? For example: 

URI Pattern: the path that a user would enter in their browser to access this route.

Don't forget about the verbs!